Frameworks/UIKit/UIView.mm

```diff
@@ -724,29 +727,34 @@ - (void)updateAccessibility {
 /**
  @Status Interoperable
 */
-+ (instancetype)allocWithZone:(NSZone*)zone {
-    UIView* ret = [super allocWithZone:zone];
-
+- (instancetype)initWithFrame:(CGRect)frame {
     // Run on the main thread because the underlying XAML objects can only be
     // called from the UI thread
     RunSynchronouslyOnMainThread(^{
```

Comment:
Maybe? Although, UIView's init calls initWithFrame, which is the designated initializer for UIView.  Regardless, I'd prefer to get this in before we make any more changes to UIView's init path (assuming this is functionally correct and isn't introducing any regressions).

---
In reply to: [87282018](https://github.com/Microsoft/WinObjC/pull/1318#discussion_r87282018) [](ancestors = 87282018)